python/tank_vendor/shotgun_authentication/ui/aspect_preserving_label.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2013 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from .qt_abstraction import QtGui
+from .qt_abstraction import QtCore
+
+
+class AspectPreservingLabel(QtGui.QLabel):
```

Comment:
not a big deal here, but i would definitely just scale the image and get rid of this code. the less code, the better IMHO.